pouchdb/find_test.go

```diff
@@ -80,7 +80,7 @@ func TestExplain(t *testing.T) {
 	tests.Add("query error", test{
 		db:    &db{db: bindings.GlobalPouchDB().New("foo", nil)},
 		query: nil,
-		err:   "TypeError: Cannot read propert",
+		err:   "TypeError: Cannot read property",
```

Comment:
This one was actually correct before, as it's a regular expression, meant to match on `property` and `properties`, I believe.  It seems that at present, with the modern versions of NodeJS and PouchDB, it's always `properties`, though.

```suggestion
		err:   "TypeError: Cannot read properties",
```

Reply:
Thnaks for the clarification, changed as requested.